worker/scraper.py

```diff
@@ -47,6 +47,22 @@ def scrape_and_categorize_articles():
         logger.info("CATEGORIZED [%s]: (%s, %s)" % (item.title, item.category_1, item.category_2))
 
 
+def scrape_feed_articles(feed_id):
+    feed = Feed.query.get(feed_id)
```

Comment:
Now that the queue is working, can we see if there's a way to pass the feed object? It would be nice to not have to do a lookup.